tests/Unit/Parallel/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ add_algorithm_test(Test_AlgorithmNodelock)
 add_algorithm_test(Test_AlgorithmReduction)
 add_algorithm_test(Test_GlobalCache)
 add_algorithm_test(Test_SectionReductions)
+add_algorithm_test(Test_CheckpointRestart)
```

Comment:
These are sort-of alphabetical, so let's not make it worse.

Reply:
Added a commit to make the entire list better (it conflicts with the fixup that moves this new line, but I'll sort that out when you ask me to squash)